DOC: xorbits.numpy getting started (#94)
xprobe-inc/xorbits: doc/source/getting_started/numpy.rst

===================================
10 minutes to :code:`xorbits.numpy`
===================================

.. currentmodule:: xorbits.numpy

This is a short introduction to :code:`xorbits.numpy` which is originated from NumPy's quickstart.

Array Creation
--------------

> Popular mechanisms for creating arrays in numpy are supported in :code:`xorbits.numpy`.

Author: ```suggestion
Popular mechanisms for creating arrays in NumPy are supported in :code:`xorbits.numpy`.
```

    ...     print(i**(1 / 3.))
    ...
    9.999999999999998
    1.0
    9.999999999999998
    3.0
    9.999999999999998
    4.999999999999999
    5.999999999999999
    6.999999999999999
    7.999999999999999
    8.999999999999998

**Multidimensional** arrays can have one index per axis. These indices
are given in a tuple separated by commas::

    >>> def f(x, y):
    ...     return 10 * x + y
    ...
    >>> b = np.arange(12).reshape(3, 4)
    >>> b
    array([[ 0,  1,  2,  3],
           [ 4,  5,  6,  7],
           [ 8,  9, 10, 11]])
    >>> b[2, 3]
    11
    >>> b[0:3, 1]  # each row in the second column of b
    array([1, 5, 9])
    >>> b[:, 1]    # equivalent to the previous example
    array([1, 5, 9])
    >>> b[1:3, :]  # each column in the second and third row of b
    array([[ 4,  5,  6,  7],
           [ 8,  9, 10, 11]])

When fewer indices are provided than the number of axes, the missing
indices are considered complete slices\ ``:``

::

    >>> b[-1]   # the last row. Equivalent to b[-1, :]
    array([ 8,  9, 10, 11])

The expression within brackets in ``b[i]`` is treated as an ``i``
followed by as many instances of ``:`` as needed to represent the

> remaining axes. NumPy also allows you to write this using dots as

Author: ```suggestion
remaining axes. You can also write this using dots as
```
